projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -240,6 +241,26 @@ private function load_value_from_callback( $task, $callback, $default = '' ) {
 		return $default;
 	}
 
+	/**
```

Comment:
Minor, but it might be a good idea to explain in the docblock why we're using a callback instead of a string.